src/FixedDataTable.js

```diff
@@ -577,9 +577,9 @@ class FixedDataTable extends React.Component {
       this._didScrollStart();
     }
 
-    // Cancel any pending debounced scroll handling and handle immediately.
```

Comment:
Welp! Hopefully there won't be any [issues](https://github.com/schrodinger/fixed-data-table-2/pull/384#discussion_r248843909) by removing didScrollStop on componentDidRecieveProps?